torchrl/modules/tensordict_module/common.py

```diff
@@ -166,7 +166,7 @@ def __init__(
         safe: bool = False,
     ):
 
-        super().__init__()
+        super(TensorDictModule, self).__init__()
```

Comment:
Why do we need this change?

Reply:
I will revert it, it think it got changed by Copilot and i didn't pay attention when saving